internal/migrate/create.go

```diff
@@ -47,6 +47,13 @@ func Create(identifier string, db DB) error {
 			return Error.Wrap(utils.CombineErrors(err, tx.Rollback()))
 		}
 
+		// creating multi column unique index with condition.
+		// status = 1 indicates that account is activated
+		_, err = tx.Exec(db.Rebind("CREATE UNIQUE INDEX console_unique_index_email_status ON users(email, status) WHERE (status != 0)"))
```

Comment:
`internal/migrate` is shared by all different databases in the system, so this is definitely not the correct place where to do this.